client/my-sites/email/form/mailboxes/validators.ts

```diff
@@ -279,12 +281,80 @@ class ExistingMailboxNamesValidator extends BaseValidator< string > {
 	}
 }
 
+class MailboxNameValidityValidator extends BaseValidator< string > {
```

Comment:
Given that we already have a `MailboxNameValidator` class and given that `ValidityValidator` seems a bit redundant, would it make sense to name this class differently?

Maybe `MailboxNameAvailabilityValidator` could be a contender?

Reply:
Sounds about right